play/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/PublishInternalSharingBundle.kt

```diff
@@ -4,39 +4,65 @@ import com.github.triplet.gradle.play.PlayPublisherExtension
 import com.github.triplet.gradle.play.tasks.internal.ArtifactExtensionOptions
 import com.github.triplet.gradle.play.tasks.internal.PublishTaskBase
 import com.github.triplet.gradle.play.tasks.internal.workers.PlayWorkerBase
+import com.github.triplet.gradle.play.tasks.internal.workers.copy
 import com.github.triplet.gradle.play.tasks.internal.workers.paramsForBase
+import org.gradle.api.file.ConfigurableFileCollection
 import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.file.RegularFileProperty
-import org.gradle.api.tasks.InputFile
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.tasks.InputFiles
 import org.gradle.api.tasks.OutputDirectory
 import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
+import org.gradle.api.tasks.SkipWhenEmpty
 import org.gradle.api.tasks.TaskAction
 import org.gradle.kotlin.dsl.submit
 import org.gradle.kotlin.dsl.support.serviceOf
 import org.gradle.workers.WorkerExecutor
+import java.io.File
 import javax.inject.Inject
 
 internal abstract class PublishInternalSharingBundle @Inject constructor(
         extension: PlayPublisherExtension
 ) : PublishTaskBase(extension), ArtifactExtensionOptions {
     @get:PathSensitive(PathSensitivity.RELATIVE)
-    @get:InputFile
-    internal abstract val bundle: RegularFileProperty
+    @get:SkipWhenEmpty
+    @get:InputFiles
+    internal abstract val bundles: ConfigurableFileCollection
```

Comment:
`PublishInternalSharingBundle` task has been updated to keep it consistent with `PublishBundle` task.